.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,29 +1,95 @@
-<!-- Please feel free to remove whatever sections/lines in this aren’t relevant.
+<!---
+Welcome! Thank you for contributing to the U.S. Web Design System. Your contributions are vital to our success and we are glad you're here.
 
-Use the title line as the title of your pull request, then delete these lines.
+Please keep in mind:
+- This pull request (PR) template exists to help speed up integration. The USWDS Core team reviews and approves every PR before merging it into the public code base, so the better we can understand the problem and solution, the sooner we can merge this change. The point here is: clear explanations matter!
+- You can erase any part of this template that doesn't apply to your pull request (including these instructions!).
 
-## Title line template: [Title]: Brief description
+If you get stuck, you can find more information about contributing in [contributing.md](https://github.com/uswds/uswds/blob/develop/CONTRIBUTING.md) or you can reach out to us directly at uswds@gsa.gov.
+ -->
 
```

Comment:
Shorten line length on all comments

Reply:
Add: Short thing about what to expect next and when to expect it.

Reply:
Shortened line length in [7988157](https://github.com/uswds/uswds/pull/4842/commits/798815701722c7167ce180b0de9c74cf7f6b3f59)